spec-main/api-app-spec.ts

```diff
@@ -195,32 +174,32 @@ describe('app module', () => {
         return
       }
 
-      const electronPath = remote.getGlobal('process').execPath
-      const appPath = path.join(__dirname, 'fixtures', 'api', 'singleton')
-      appProcess = ChildProcess.spawn(electronPath, [appPath])
+      const electronPath = process.execPath
+      const appPath = path.join(fixturesPath, 'api', 'singleton')
+      appProcess = cp.spawn(electronPath, [appPath])
 
       // Singleton will send us greeting data to let us know it's running.
       // After that, ask it to exit gracefully and confirm that it does.
-      appProcess.stdout.on('data', data => appProcess.kill())
+      appProcess.stdout.on('data', data => appProcess && appProcess.kill())
```

Comment:
how would `appProcess` be null here? should this be
```suggestion
      appProcess.stdout.on('data', data => appProcess!.kill())
```